tools/manifest/manifest.py

```diff
@@ -281,18 +281,20 @@ def update(self, tree):
                         hash_changed = True
                         if new_type != old_type:
                             del self._data[old_type][rel_path]
+                            if old_type in reftest_types:
+                                reftest_changes = True
                     else:
-                        new_type, manifest_items = old_type, self._data[old_type][rel_path]
-                    if old_type in reftest_types and new_type != old_type:
-                        reftest_changes = True
+                        new_type = old_type
+                        if old_type in reftest_types:
+                            manifest_items = self._data[old_type][rel_path]
```

Comment:
To be clear I assume it's OK to not set `reftest_changes` here since the check below under `new_type in reftest_types` covers all the relevant changes?

Reply:
Dunno. But we didn't do it before this patch either (given in this case `new_type == old_type`).

Reply:
Oh, also this is the "no changes" case, so there aren't any `reftest_changes` here.